packages/integration-tests/projects/suite-web/tests/suite/safety-checks-warning.test.ts

```diff
@@ -29,38 +29,62 @@ describe('safety_checks Warning For PromptTemporarily', () => {
         // Start in the device settings to easily open safety_checks setting modal.
         cy.prefixedVisit('/');
         cy.passThroughInitialRun();
+        cy.discoveryShouldFinish();
         cy.getTestElement('@suite/menu/settings').click();
         cy.getTestElement('@suite/menu/settings-index').click();
         cy.getTestElement('@settings/menu/device').click();
 
         // Set safety_checks to `PromptTemporarily'.
         // TODO: do this via the `applySetting` task once https://github.com/trezor/trezor-user-env/issues/54
         // is resolved.
-        cy.getTestElement('@settings/device/safety-checks-button').click();
-        cy.get(`[data-test="@radio-button-prompt"]`).click();
-        cy.getTestElement('@safety-checks-apply').click();
-        cy.task('pressYes');
     });
 
     it('Dismissible warning appears', () => {
+        cy.getTestElement('@settings/device/safety-checks-button').click({
+            scrollBehavior: 'bottom',
+        });
+        cy.get(`[data-test="@radio-button-prompt"]`).click();
+        cy.getTestElement('@safety-checks-apply').click();
+        cy.task('pressYes');
         cy.getTestElement('@banner/safety-checks/button');
         cy.getTestElement('@banner/safety-checks/dismiss');
     });
 
     it('CTA button opens device settings', () => {
+        cy.getTestElement('@settings/device/safety-checks-button').click({
+            scrollBehavior: 'bottom',
+        });
+        cy.get(`[data-test="@radio-button-prompt"]`).click();
+        cy.getTestElement('@safety-checks-apply').click();
+        cy.task('pressYes');
         cy.getTestElement('@banner/safety-checks/button').click();
         // In CI the path is prefixed with a branch name. Test only against the end of the path.
         cy.location('pathname').should('match', /\/settings\/device\/$/);
     });
 
     it('Dismiss button hides the warning', () => {
+        cy.getTestElement('@settings/device/safety-checks-button').click({
+            scrollBehavior: 'bottom',
+        });
+        cy.get(`[data-test="@radio-button-prompt"]`).click();
+        cy.getTestElement('@safety-checks-apply').click();
+        cy.task('pressYes');
         cy.getTestElement('@banner/safety-checks/dismiss').click();
         cy.getTestElement('@banner/safety-checks/button').should('not.exist');
     });
 
     it('Warning disappears when safety_checks are set to strict', () => {
         // Open the safety_checks setting modal and change safety_checks to Strict.
```

Comment:
This comment is now out of context. PLease, move it accordingly.

Reply:
9fb2d0acca19cadd8ec33eb32777c2fdadf321be